doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md

```diff
@@ -0,0 +1,89 @@
+# ADR-0001: Update the InVEST SDR LS Factor
+
+Author: James
```

Comment:
need some trailing spaces for newline here

Reply:
Patched in [8e2b41e](https://github.com/natcap/invest/pull/1288/commits/8e2b41e8542ac24f69092f32a30323c3e7f92f61)